packages/api-gateway-service/service.ts

```diff
@@ -0,0 +1,82 @@
+import express from 'express';
+import { ApolloServer } from 'apollo-server-express';
+import fs from 'fs';
+import https from 'https';
+import http from 'http';
+import { ApolloGateway } from '@apollo/gateway';
+import cookieParser = require( 'cookie-parser' );
+import { serviceList } from './src/helpers';
+const { ApolloLogExtension } = require( 'apollo-log' );
+
+import { verify } from 'jsonwebtoken';
+
+/* Setting port for the server */
+const port = process.env.PORT || 4000;
+const app = express();
+
+/* Mount cookie parker */
+app.use( cookieParser() );
+
+/* Auth Token Verification */
+// app.use( ( req, res, next ) => {
+//   publicKey().then( ( key: string ) => {
+//     console.log( key );
+//     const accessToken = req.cookies[ 'access-token' ];
+//     if ( !accessToken ) {
+//       return next();
+//     }
+
+//     try {
+//       const data = verify( accessToken, key ) as any;
+//       console.log( accessToken, data );
+//       return next();
+//     } catch { }
+//   } );
+//   next();
+// } );
```

Comment:
Remove this block from this PR. When it's complete and implemented, you can send a different PR for it.

Reply:
removed this code